pkg/network/vlan/vlan.go

```diff
@@ -251,15 +243,37 @@ func (v *Vlan) afterLinkDown(update netlink.LinkUpdate) {
 			Reason:    "LinkDown",
 			Message:   fmt.Sprintf("Link %s has been down atypically and the controller will try to set it up", update.Link.Attrs().Name),
 		}
-
 		if v.helper != nil && v.helper.EventSender != nil {
 			if err := v.helper.EventSender(event, v.Type()); err != nil {
 				klog.Errorf("recorde event failed, error: %s", err.Error())
+				return
```

Comment:
What happens when it returns here? Will it do another try to recover routes?

Reply:
No

Reply:
I will delete `return` sentence to let it continue to try to set the down link up or sync IPv4 addresses.